src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
```

Comment:
Similarly, the message 'Excavator stopped working.' could be more informative. Consider specifying what has stopped, for instance: 'Excavator has stopped working.'